src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsrestrictedpatientsapi/controllers/RestrictedPatientsBatchController.kt

```diff
@@ -29,8 +30,7 @@ class RestrictedPatientsBatchController(
   @PreAuthorize("hasRole('RESTRICTED_PATIENT_MIGRATION')")
   /**
    * Internal endpoint to migrate unknown patients into Nomis and add them to Restricted Patients
-   * TODO SDI-357 This is still a WIP
    */
-  fun processUnknownPatients(@RequestBody patients: List<String>): List<UnknownPatientResult> =
-    unknownPatientService.migrateInUnknownPatients(patients)
+  fun processUnknownPatients(@RequestBody patients: List<String>, @RequestParam(required = false, defaultValue = "true") dryRun: Boolean): List<UnknownPatientResult> =
```

Comment:
a `/dryrun-unknown-patients` endpoint might be clearer and less confusing as to what the endpoint does when the parameter isn't supplied